### ISSUE TITLE
experimental results validation or test?

### ISSUE THREAD

**Issue by hahahaa1** [mkang315/RCS-YOLO#13]
I would like to ask that in the author's paper, it was mentioned that the Br35H dataset was divided into a training set and a validation set, but there was no mention of a test set. I have searched for the Br35H dataset on the Internet, and it contains a training set, a validation set, and a test set. I would like to inquire whether the final experimental results obtained by the author in the paper were obtained on the validation set or the test set?

**Comment by mkang315** [comment]
As stated in Section 3.1, our experimental results of RCS-YOLO are obtained on the validation set. Please see the results of RCS-YOLO on the test set in the BGF-YOLO's [paper](https://arxiv.org/pdf/2309.12585), and its code is available at https://github.com/mkang315/BGF-YOLO. Thank you!
